examples/deno/index.ts

```diff
@@ -1,10 +1,10 @@
-import { serve } from 'https://deno.land/std@0.117.0/http/server.ts'
+import { serve } from 'https://deno.land/std@0.153.0/http/server.ts'
 import { createServer } from 'https://cdn.skypack.dev/@graphql-yoga/common'
 
 const graphQLServer = createServer()
 
-serve(graphQLServer, {
```

Comment:
I'll check why this doesn't work.

Reply:
it does not work because `serve` accepts a handler function and `graphqlServer` is an object.

`serve(handler: Handler, options?: ServeInit): Promise<void>`

Reply:
`graphqlServer` is a callable object so it is technically a function, but it seems it doesn't accept the signature. I'll push a fix for that :)